pk3DS/Subforms/Gen7/TutorEditor7.cs

```diff
@@ -99,5 +99,35 @@ private void B_Randomize_Click(object sender, EventArgs e)
         {
             WinFormsUtil.Alert("Not currently implemented.");
         }
+
+        internal static int[][] getTutorList()
+        {
+            string[] locationsTutor =
+            {
+                "Big Wave Beach",
+                "Heahea Beach",
+                "Ula'ula Beach",
+                "Battle Tree"
+            };
+            
+            byte[] data = File.ReadAllBytes(Path.Combine(Main.RomFSPath, "Shop.cro"));
+            byte[] len_BPTutor = data.Skip(0x52D2).Take(4).ToArray();
```

Comment:
Using LINQ for byte operations like this is rather slow. Array.Copy is preferable.